docs/app/pages/components/components-sections/splitter/side-inset-panel-splitter/side-inset-panel-splitter.component.less

```diff
@@ -7,4 +7,10 @@
 .splitter-pane {
     position: relative;
     padding: 30px 10px 10px;
+}
+
+as-split {
+    ::ng-deep.as-split-gutter {
+        background-color: #f2f2f2 !important;
```

Comment:
Looks like this doesn't work. Also this should be part of the component style, not the documentation.

Reply:
updated